src/app/account-details/personal-details.component.ts

```diff
@@ -60,6 +61,7 @@ export class PersonalDetailsComponent {
     ) {
         this.details.subscribe((details: AccountDetailsInterface) => {
             this.detailsForm.patchValue(details);
+            this.is_alternative_email_validated = details.email_alternative_status == 0;
```

Comment:
The triple equal has better equality check characteristics. I only use == for nil (null/undefined) checks. If you want to just check if something is falsy you can also do a bang: `!details.email_alternative_state`.